datasets/pcfg/data_generation.py

```diff
@@ -149,26 +148,31 @@ def main(argv: Sequence[str]):
     
     out_file_src = open(data_path + 'it_dec_val.src', 'w')
     out_file_tgt = open(data_path + 'it_dec_val.tgt', 'w')
-    
     out_file_src_dec = open(data_path + 'it_dec_test.src', 'w')
     out_file_tgt_dec = open(data_path + 'it_dec_test.tgt', 'w')
+    ops_file = open(data_path + 'it_dec_test.ops', 'w')
```

Comment:
Similarly, "ops_file" is only for test, so maybe call it "out_file_ops_test", to match with the other naming scheme?